src/strategies/eth-wallet-age/index.ts

```diff
@@ -0,0 +1,45 @@
+import fetch from 'cross-fetch';
+
+export const author = 'chaituvr';
+export const version = '0.1.0';
+
+const delay = interval => new Promise(resolve => setTimeout(resolve, interval));
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+  ) {
+  const apiLimit = 300;
+  let data = []
+
+  data = await Promise.all(addresses.map((address, index) => {
+    return new Promise(resolve => setTimeout(resolve, apiLimit * index))
```

Comment:
We can't send a request for each voter, some proposal have 1000 votes, we dont want the user browser to send 1000 requests. Not sure how you can bypass this.

Reply:
Maybe using an API like the one used here can work https://github.com/snapshot-labs/snapshot.js/pull/60